content/en/docs/setup/independent/high-availability.md

```diff
@@ -326,67 +342,70 @@ done
     - `CP2_HOSTNAME`
     - `CP2_IP`
 
-1. Move the copied certificates to the proper locations
-
-  ```sh
-  USER=ubuntu # customizable
-  mkdir -p /etc/kubernetes/pki/etcd
-  mv /home/${USER}/ca.crt /etc/kubernetes/pki/
-  mv /home/${USER}/ca.key /etc/kubernetes/pki/
-  mv /home/${USER}/sa.pub /etc/kubernetes/pki/
-  mv /home/${USER}/sa.key /etc/kubernetes/pki/
-  mv /home/${USER}/front-proxy-ca.crt /etc/kubernetes/pki/
-  mv /home/${USER}/front-proxy-ca.key /etc/kubernetes/pki/
-  mv /home/${USER}/etcd-ca.crt /etc/kubernetes/pki/etcd/ca.crt
-  mv /home/${USER}/etcd-ca.key /etc/kubernetes/pki/etcd/ca.key
-  mv /home/${USER}/admin.conf /etc/kubernetes/admin.conf
-  ```
-
-1. Run the kubeadm phase commands to bootstrap the kubelet
-
-  ```sh
-  kubeadm alpha phase certs all --config kubeadm-config.yaml
-  kubeadm alpha phase kubelet config write-to-disk --config kubeadm-config.yaml
-  kubeadm alpha phase kubelet write-env-file --config kubeadm-config.yaml
-  kubeadm alpha phase kubeconfig kubelet --config kubeadm-config.yaml
-  systemctl start kubelet
-  ```
+1. Move the copied certificates to the proper locations:
+
```

Comment:
Urgh, note to self s/proper/correct

For reviewers: is there a reason we provide the copying information this way here, but in a loop script ^^? Consistency is a good thing in techdoc, if it's appropriate here ...